panel/pane/vtk/synchronizable_serializer.py

```diff
@@ -566,11 +630,24 @@ def genericProp3DSerializer(parent, prop3D, prop3DId, context, depth):
 
 
 def genericActorSerializer(parent, actor, actorId, context, depth):
-    # may have texture and 
+    # may have texture and
     instance = genericProp3DSerializer(parent, actor, actorId, context, depth)
 
     if not instance: return
 
+    # # actor may have a backface property instance (not used by vtkjs rendering)
+    # # https://github.com/Kitware/vtk-js/issues/1545
+
+    # backfaceProperties = actor.GetBackfaceProperty()
+
+    # if backfaceProperties:
+    #     backfacePropId = context.getReferenceId(backfaceProperties)
+    #     backPropertyInstance = serializeInstance(
+    #         actor, backfaceProperties, backfacePropId, context, depth + 1)
+    #     if backPropertyInstance:
+    #         instance['dependencies'].append(backPropertyInstance)
+    #         instance['calls'].append(['setBackfaceProperty', [wrapId(backfacePropId)]])
+
```

Comment:
Keep this?

Reply:
Yes we can keep this if vtkjs is going to support backface color we won't have to rethink about it